source/common/grpc/grpc_web_filter.h

```diff
@@ -14,7 +17,7 @@ namespace Grpc {
  */
 class GrpcWebFilter : public Http::StreamFilter, NonCopyable {
```

Comment:
not really related to this change, but can you look at coverage for this filter in master and make sure this commit gets it up to 100% coverage. I think you need to call onDestroy() filter method in your test fixture at minimum.